posts/oversubscribed-tracks.md

```diff
@@ -0,0 +1,13 @@
+One of the biggest challenges we face is when tracks get too popular for our mentors to keep up. The wait time for students increases, and mentors have the very unsatisfying experience of seeing queues keep growing despite putting a huge amount of work in to try and keep on top.
```

Comment:
```suggestion
One of the biggest challenges we face is tracks getting to popular for our mentors to keep up. The wait time for students increases, and mentors have the very unsatisfying experience of seeing queues keep growing despite putting a huge amount of work in to try and keep on top.
```

The `is when` phrasing sounds a bit clunky to me, though I'm not a native English speaker so maybe that's why.